antenna-frontend-stubs/cli-frontend-stub/src/main/java/org/eclipse/sw360/antenna/frontend/cli/AntennaCLISettingsReader.java

```diff
@@ -101,17 +102,10 @@ ToolConfiguration readSettingsToToolConfiguration(DefaultProject project) {
         return readSettingsToToolConfiguration(reader, project);
     }
 
-    private void readAntennaWorkflowSetting(XmlSettingsReader reader, String name, Consumer<Workflow> setter) {
-        Workflow workflow = reader.getComplexType("workflow", Workflow.class);
-        if (workflow != null) {
-            setter.accept(workflow);
-        }
-    }
-
     private void setVersionFromPom(DefaultProject project) {
         File pomFile = project.getConfigFile();
         try {
-            String pom = FileUtils.readFileToString(pomFile);
+            String pom = FileUtils.readFileToString(pomFile, Charset.forName("UTF-8"));
```

Comment:
This avoids spelling errors. If I misspell "UTF-8", the code won''t compile, otherwise it will throw an error at runtime. Also, this method is the preferred replacement for the deprecated method with one parameter.